### PR TITLE
system tests must pass

### PR DESCRIPTION
**Pull request by baude**
Signed-off-by: Brent Baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

File: cmd/podman/common/util.go [hunk 1]
```diff
@@ -1,8 +1,11 @@
 package common
 
 import (
+	"fmt"
 	"strconv"
 
+	"github.com/spf13/cobra"
+
```
Comment (jwhonce) [inline cmd/podman/common/util.go:8]:
```suggestion
```

File: cmd/podman/common/util.go [hunk 2]
```diff
@@ -41,3 +44,11 @@ func createPortBindings(ports []string) ([]ocicni.PortMapping, error) {
...
 	return portBindings, nil
 }
+
+// NoArgs returns an error if any args are included.
+func NoArgs(cmd *cobra.Command, args []string) error {
+	if len(args) > 0 {
+		return fmt.Errorf("`%s` takes no arguments", cmd.CommandPath())
+	}
+	return nil
+}
```
Comment (jwhonce) [inline cmd/podman/common/util.go:54]:
```suggestion
// cobra.ExactArgs(0)
}
```

File: cmd/podman/containers/ps.go [hunk 3]
```diff
@@ -25,7 +26,7 @@ var (
 	psDescription = "Prints out information about the containers"
 	psCommand     = &cobra.Command{
 		Use:   "ps",
-		Args:  checkFlags,
+		Args:  common.NoArgs,
```
Comment (jwhonce) [inline cmd/podman/containers/ps.go:29]:
```suggestion
                Args: func(cmd *cobra.Command, args []string) error {
                   err := cobra.ExactArgs(1)(cmd, args)
                   if err != nil {
                       return err
                   }
                   return checkFlags(cmd, args)
               }
```

File: cmd/podman/containers/ps.go [hunk 4]
```diff
@@ -141,6 +142,9 @@ func getResponses() ([]entities.ListContainer, error) {
 
 func ps(cmd *cobra.Command, args []string) error {
 	var responses []psReporter
+	if err := checkFlags(cmd, args); err != nil {
+		return err
+	}
```
Comment (jwhonce) [inline cmd/podman/containers/ps.go:147]:
```suggestion
```

File: cmd/podman/manifest/add.go [hunk 5]
```diff
@@ -13,7 +13,7 @@ import (
 var (
 	manifestAddOpts = entities.ManifestAddOptions{}
 	addCmd          = &cobra.Command{
-		Use:   "add",
+		Use:   "add [flags] LIST LIST",
```
Comment (jwhonce) [inline cmd/podman/manifest/add.go:16]:
```suggestion
		Use:   "add [flags] LIST LIST",
```
Is `LIST LIST` correct? Shouldn't one be IMAGE?

File: cmd/podman/manifest/create.go [hunk 6]
```diff
@@ -13,7 +13,7 @@ import (
 var (
 	manifestCreateOpts = entities.ManifestCreateOptions{}
 	createCmd          = &cobra.Command{
-		Use:   "create",
+		Use:   "create [flags] LIST [IMAGE]",
```
Comment (jwhonce) [inline cmd/podman/manifest/create.go:16]:
```suggestion
		Use:   "create [FLAGS] LIST [IMAGE]",
```
Some commands are [flag] others [FLAG] we should be consistent.

File: cmd/podman/containers/exists.go [hunk 7]
```diff
@@ -17,8 +17,9 @@ var (
 		Long:  containerExistsDescription,
 		Example: `podman container exists containerID
   podman container exists myctr || podman run --name myctr [etc...]`,
-		RunE: exists,
-		Args: cobra.ExactArgs(1),
+		RunE:                  exists,
+		Args:                  cobra.ExactArgs(1),
+		DisableFlagsInUseLine: true,
```
Comment (jwhonce) [inline cmd/podman/containers/exists.go:22]: I think this should be automatically set in main.go

**Pull request merged**